dotcom-rendering/src/components/Card/components/ImageWrapper.tsx

```diff
@@ -137,7 +137,9 @@ export const ImageWrapper = ({
 					flexBasisStyles({
 						imageSize,
 					}),
-				(imageType === 'picture' || imageType === 'video') &&
+				(imageType === 'picture' ||
+					imageType === 'video' ||
+					imageType === 'podcast') &&
```

Comment:
The conditional immediately before this seems to be applying exactly the same same styles just for slideshows so could be added to this?

```jsx
imageType === 'slideshow' &&
	isHorizontalOnDesktop &&
	flexBasisStyles({
		imageSize,
	}),
```
					}),
```

Reply:
good spot! ive refactored for this.